scala3/getting-started.md

```diff
@@ -123,20 +123,20 @@ They both offer rich IDE features, but you can still use [many other editors.](h
 
 View these two files in your IDE:
 
-- *build.sbt*
-- *src/main/scala/Main.scala*
+- _build.sbt_
+- _src/main/scala/Main.scala_
```

Comment:
Typically, I would also use inline code for file names (that is, `build.sbt`). But I have seen you using this style elsewhere, so I guess it is ok.

Reply:
Yes, this is a place where I use the O’Reilly style. They use italics for filenames, directory names, and URIs/URLs.